spec-main/api-app-spec.ts

```diff
@@ -876,6 +876,32 @@ describe('app module', () => {
         })
       })
     })
+
+    it('sets the default client such that getApplicationNameForProtocol returns Electron', (done) => {
+      app.setAsDefaultProtocolClient(protocol);
+      expect(app.getApplicationNameForProtocol(`${protocol}://`)).to.equal('Electron')
+      done()
```

Comment:
if this isn't an explicitly async new method call (docs don't indicate it is unless i missed something) we probably don't need this `done` here